### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ludwigbothmann**
update citation

### CODE REVIEW

File: R/summary.R [hunk 1]
```diff
@@ -440,7 +440,7 @@ summary.Graph = function(object, resample_result = NULL, control = summary_contr
 #' @param importance_measures (character()|NULL)\cr
 #'   vector of importance measure names. Possible choices are "pfi.<loss>"
 #'   ([iml::FeatureImp]), "pdp" ([iml::FeatureEffects], see ) and
-#'   "shap" ([fastshap::explain]). Default of NULL results in "pfi.<loss> and
```
Comment (ludwigbothmann) [inline R/summary.R:443]: The "<loss>" does not render for me, do you know why?

Comment (ludwigbothmann) [reply R/summary.R:443]: Haha, also not in my comment :) it is about the smaller-than-loss-greater-than signs

File: README.md [hunk 2]
```diff
@@ -52,10 +53,22 @@ summary(object = rf, resample_result = rr)
...
-toBibtex(citation("mlr3summary"))
+
+```
+Dandl S, Becker M, Bischl B, Casalicchio G, Bothmann L (2024).
+mlr3summary: Model and learner summaries for 'mlr3'.
+R package version 0.1.0.
 ```
+A BibTeX entry for LaTeX users is
 
+```
```
Comment (ludwigbothmann) [inline README.md:67]: I like this better because you can directly copy-paste it

File: R/summary.R (hunk 1 above)
Comment (dandls) [reply R/summary.R:443]: Thanks. I know how to fix this :-) a ` must be used

**Review by dandls — APPROVED** [review]
Thanks, I merge all changes :+1:

**Pull request merged**